trie/trie_test.go

```diff
@@ -482,11 +489,11 @@ func Test2kEntries(t *testing.T) {
 	require.Nil(tr.Start(context.Background()))
 	root := EmptyRoot
 	seed := time.Now().Nanosecond()
-	// insert 2k entries
+	// insert 4k entries
```

Comment:
in the same amount of time, can test 4k entries. meaning protobuf is 2x faster than Gob